docs/governance.rst

```diff
@@ -0,0 +1,253 @@
+  resolve conflicts and increase the pace of the development when necessary.
+  Any maintainer can self-remove themselves. Any contributor can become a
+  maintainer by request and with the support of the majority of the current
+  maintainers. Current Maintainers:
+
+  +-----------------------------------+-----------------+
+  | Name                              | Time commitment |
+  +===================================+=================+
+  | `Logan Dowdle`_ (@dowdlelt)       | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Elizabeth DuPre`_ (@emdupre)     | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Dan Handwerker`_ (@handwerkerd)  | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Ross Markello`_ (@rmarkello)     | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Taylor Salo`_ (@tsalo)           | 3h/week         |
+  +-----------------------------------+-----------------+
+  | `Joshua Teves`_ (@jbteves)        | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Eneko Uruñuela`_ (@eurunuela)    | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Kirstie Whitaker`_ (@KirstieJane)| 0.5h/week       |
+  +-----------------------------------+-----------------+
+
+
+Rules
+`````
+
+1. Potential modifications to the Repository should first be proposed via an 
+   Issue. Rules regarding Votes apply to both Pull Requests and Issues.
+
+   - Every modification of the specification (including a correction of a typo, adding a new Contributor, an extension adding support for a new data type, or others) or proposal to release a new version needs to be done via a Pull Request (PR) to the Repository.
+2. Anyone can open a PR (this action is not limited to Contributors).
+3. PRs adding new Contributors must also add their GitHub names to the 
+   `all-contributors file`_. 
+   This should be done with the allcontributors bot.
+
+   - Contributors may also add themselves to the Zenodo file if they wish, but this is not mandatory.
+4. A PR is eligible to be merged if and only if these conditions are met:
+
+   a) The PR features at least two `Reviews that Approve <https://help.github.com/articles/about-pull-request-reviews/#about-pull-request-reviews>`_
+      the PR from Maintainers of which neither is the author of the PR. 
+      The reviews need to be made after the last commit in the PR (equivalent to 
+      `Stale review dismissal <https://help.github.com/articles/enabling-required-reviews-for-pull-requests/>`_
+      option on GitHub).
+   b) Does not feature any `Reviews that Request changes <https://help.github.com/articles/about-required-reviews-for-pull-requests/>`_.
+   c) Does not feature "WIP" in the title (Work in Progress).
+   d) Passes all automated tests.
+   e) Is not proposing a new release or has been approved by at least one
```

Comment:
I would think only steering committee members can cut releases ? WDYT ?

Reply:
I agree @emdupre

Reply:
Resolved as of latest commit.